### DISCUSSION TITLE
ibax  listelenme hangi borsa  benim gibi büyük borsalarda listelenir diyen varmı gelsin

### DISCUSSION THREAD

**Discussion by pinarati** [IBAX-io/go-ibax#277] [Ecolibs]
ibax  hangi borasada listelenir bence büyük borsalar kapar

**Comment by byblckd** [comment]
önce bir ön satış için platform ile anlaşssın listeleme işine bakarız:)

**Comment by pinarati** [comment]
hadi inşallah ben umutluyum
